task-grid.jade

```diff
@@ -0,0 +1,3 @@
+.row
+  div(ng-controller="TaskGridController", style="background:white")
+    div(ui-grid="gridOptions" ui-grid-pagination style="width:100%;")
```

Comment:
please, use commas to separate attrs on the same line. And inline styles is a bad thing, use css